tests/unit/jobProcessor/jobProcessor.spec.ts

```diff
@@ -1,13 +1,33 @@
 import nock from 'nock';
 import { JobProcessor } from '../../../src/models/jobProcessor';
 import { configMock, registerDefaultConfig } from '../mocks/configMock';
-import { jobProcessorInstace } from '../jobProcessor/jobProcessorSetup';
+import { invalidJobResponseMock, newJobResponseMock } from '../mocks/jobsMocks';
+import { initTaskForIngestionNew } from '../mocks/tasksMocks';
+
+const initJobHandlerMock = jest.fn();
+
+initJobHandlerMock.mockImplementation(() => {
+  return {
+    processJob: async () => {
+      await mockProcessJob();
+    },
+  };
+});
+
+jest.mock<typeof import('../../../src/models/handlersFactory')>('../../../src/models/handlersFactory', () => {
```

Comment:
double check how can you make it more readable please:
<typeof import('../../../src/models/handlersFactory')>